NAMESPACE

```diff
@@ -25,6 +25,7 @@ export(mmrm_control)
 export(refit_multiple_optimizers)
 import(checkmate)
 importFrom(Rdpack,reprompt)
+importFrom(lifecycle,deprecate_warn)
```

Comment:
no need to import this - just use the `::` operator. we just always import a single function (I think that is also in the dev guide)

Reply:
If so I didn't see it. What do you mean with importing a single function? My reference would be https://kbroman.org/pkg_primer/pages/depends.html since we do not use a lot of the lifecycle package @importFrom is clearer and makes it easier to clean up the namespace after removing the deprecation has been removed. Anyway, not really important to me - @import or @importFrom?

Reply:
Hm ok we need to add that to the contributing guide.
Basically no need to import any more here. Just use `lifecycle::deprecate_warn()` in the code.

Reply:
@danielinteractive, Can we have a description / comment on this choice? We use `lifecycle::` in the header but don't mention this? Is this OK mainly because `.Rd` are compiled by hand, separately from the functioning of the package?

Reply:
@BrianLang yeah we will add this to the dev guide